GameOfLifeApp/GameLogic.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLifeApp
+{
+    public class GameLogic
+    {
+        bool[,] generatedField;
+
+        /// <summary>
+        /// Holds user data of array size.    
+        /// </summary>
+        public int countOfRows { get; set; }
+        public int countOfColumns { get; set; }
+        public bool[,] currentField { get; set; }
+
+        /// <summary>
+        /// Gets data from user.
+        /// </summary>
+        public GameLogic()
+        {
+            countOfRows = Communicator.GetBoundedResponse("Please input number of rows.", 2, 100);
```

Comment:
A constructor should do as little as possible. Its okay to initialize default values for your game field array, but asking for user input can cause potential issues with things like exception handling, testing and mocking. Best case scenario, the column and row variables would be already passed to this constructor as parameters and in that case, you would already know the necessary values and there would be no need to ask user for user inputs.

Reply:
Done.